### PR TITLE
polyfrost

### PR DESCRIPTION
**Pull request by Mqisty**
waiting for wyvest to confirm this is how he wants it since hes lazy and balding

### CODE REVIEW

**Comment by KTibow** [comment]
fix the discord name and description and stuff

**Comment by Mqisty** [comment]
im not sure how he wants it and he ignored my dm but whatever

File: files/discords.json [hunk 1]
```diff
@@ -105,16 +105,17 @@
...
-        "description": "A server to discuss / get support for W-OVERFLOW mods and other things!",
+        "description": "A server to discuss / get support for Polyfrost mods and other things!",
         "icon": "wyvest.png",
         "nicknames": [
             "wyvest",
             "wyvtilities",
             "www",
-            "qalcyo"
+            "qalcyo",
+            "polyfrost"
```
Comment (Wyvest) [inline files/discords.json]: what lmao

**Review by Wyvest — CHANGES_REQUESTED** [review]

File: files/discords.json (hunk 1 above)
Comment (Mqisty) [reply files/discords.json]: oh shit i forgot to change that back

do you want me to add woverflow as a nickname?

**Pull request merged**